app/src/main/java/com/jtmcn/archwiki/viewer/MainActivity.java

```diff
@@ -163,20 +165,23 @@ public void hideSearchView() {
 
 	@Override
 	public boolean onCreateOptionsMenu(Menu menu) {
-		getMenuInflater().inflate(R.menu.menu, menu);
-		MenuItem share = menu.findItem(R.id.menu_share);
-		shareActionProvider = (ShareActionProvider) MenuItemCompat.getActionProvider(share);
+		MenuInflater inflater = getMenuInflater();
+		inflater.inflate(R.menu.menu, menu);
 		return true;
 	}
 
 	@Override
-	public boolean onOptionsItemSelected(MenuItem item) {
+	public boolean onOptionsItemSelected(@NonNull MenuItem item) {
 		switch (item.getItemId()) {
 			case R.id.menu_share:
 				WikiPage wikiPage = wikiViewer.getCurrentWebPage();
 				if (wikiPage != null) {
-					Intent intent = AndroidUtils.shareText(wikiPage.getPageTitle(), wikiPage.getPageUrl(), this);
-					shareActionProvider.setShareIntent(intent);
+					Intent sharingIntent = new Intent();
+					sharingIntent.setType(TEXT_PLAIN_MIME);
+					sharingIntent.setAction(Intent.ACTION_SEND);
+					sharingIntent.putExtra(Intent.EXTRA_TITLE, wikiPage.getPageTitle());
```

Comment:
I'm not sure if this is actually doing anything, I don't see it locally, but I guess we can just keep it for now